includes/Core/Util/Requires_Javascript_Trait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Core\Util\Requires_Javascript_Trait
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Trait to display no javascript fallback message.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+trait Requires_Javascript_Trait {
+
+	/**
+	 * Display fallback message when Javascript is disabled
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string noscript HTML tag,
+	 */
+	protected function get_noscript_html() {
+
+		ob_start();
+
+		?>
+			<noscript>
+				<div class="googlesitekit-noscript googlesitekit-plugin notice notice-warning">
+					<div class="mdc-layout-grid">
+						<div class="mdc-layout-grid__inner">
+							<div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-12">
+								<h3 class="googlesitekit-heading-3 googlesitekit-noscript__title">
+									<?php
+										echo esc_html__( 'The Site Kit by Google plugin requires JavaScript to be enabled in your browser.', 'google-site-kit' )
```

Comment:
Let's use the dedicated core function for this:
```suggestion
										esc_html_e( 'The Site Kit by Google plugin requires JavaScript to be enabled in your browser.', 'google-site-kit' )
```